src/frontend/sharedComponents/ActionTab/KeyboardAccessory.tsx

```diff
@@ -0,0 +1,47 @@
+import React, {FC} from 'react';
+import {View, Text, Pressable, StyleSheet} from 'react-native';
+import {NEW_DARK_GREY} from '../../lib/styles';
+import Plus from '../../images/redesign/Plus.svg';
+import {ActionTabItems} from './ActionTab';
+
+interface KeyboardAccessory {
+  items: ActionTabItems[];
+  onPress: () => unknown;
+}
+export const KeyboardAccessory: FC<KeyboardAccessory> = ({onPress, items}) => (
+  <View style={[styles.flexRow, styles.containerPadding]}>
+    <View style={[styles.flexRow, styles.accessoryMainWrapper]}>
+      <Plus style={{marginRight: 10}} />
+      <Text style={styles.text}>Show Options</Text>
```

Comment:
this text is seeping into the icons on my device. TO fix this add `flex:1` to `styles.flexRow` and then in this text add `numberOfLines={1}`

Reply:
Thanks for nit about that.
I fixed that, and now that should work properly.